### DISCUSSION TITLE
SGI netboot IRIX install

### DISCUSSION THREAD

**Discussion by tanuki2k** [retronas/discussions#61] [Ideas]
Any plans for SGI netboot for installing IRIX? I know there are project like DINA but I believe SGI systems just use bootp, TFTP and RSH to install the OS over the network.

**Comment by danmons** [comment]
We're definitely interested in netbooting.  @sairuk has been playing with some Mac stuff, and I've got some DHCP/BOOTP stuff I've been mucking about with for a bunch of OSes (along with the existing TFTP stuff we have, and iPXE+HTTP for newer x86 systems). 

I'm not familiar with SGI IRIX netbooting specifically, but if we can get an emulator up and running in MAME or something, perhaps we can get a test environment going.

**Comment by tanuki2k** [comment]
I found this guide on IRIX and [MAME](https://sgi.neocities.org/installguide.html) and here is a page on [DINA](http://shiftleft.com/mirrors/dina.harrydebug.com/) and [irixboot](https://github.com/halfmanhalftaco/irixboot). I even found a guide setting up [PI](http://www.cs.columbia.edu/~sedwards/presentations/2019-vcf-netboot.pdf) for IRIX and sun computers. I hope that helps.
